service/liquidator/scripts/intent-swap.sh

```
@@ -50,7 +50,7 @@ SLIPPAGE_BPS="300"
 REFERRAL="templar-manual"
 API_BASE_URL="https://1click.chaindefuser.com"
 NETWORK="mainnet"
-RPC_URL="https://free.rpc.fastnear.com"
+NEAR_RPC_URL="https://free.rpc.fastnear.com"
```

Shellcheck (0.11.0)

[warning] NEAR_RPC_URL appears unused. Verify use (or export if used externally).

(SC2034)